recipes/arb-bio/6.1.rev16223/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "ARB" %}
+{% set version = "6.1.rev16223" %}
+
+package:
+  name: {{ name|lower }}-bio
+  version: {{ version }}
+
+source:
+  svn_url: http://vc.arb-home.de/readonly/trunk
+  svn_rev: 16223
+  sha256: none
+  patches:
+    # do not create links for mafft
+   - no_mafftlinks.patch
+   # allow linking with missing symbols on macos
+   - link_dyn.patch
+   - make.patch
+   # change all /usr/bin/xyz paths to /usr/bin/env xyz
+   - abs_path_fix.patch
+   # use sed, not gsed
+   - gsed.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ccache-toolchain
+    - gcc   # [linux]
+    - llvm  # [osx]
+    - pkg-config
+    - openmotif-dev
+    - glib 2.51.* # 2.50.2 links to broken gettext
+    - gettext !=0.19.8  # gettext 0.19.8 is broken
+    - pthread-stubs
+    - xorg-libxp
+    - xorg-libxmu
+    - xorg-libxi
+    - xorg-libxaw
+    - xorg-libxpm
+    - xorg-libxft
+    - xorg-makedepend
+    - xerces-c
+    - libtiff
```

Comment:
The only one out of that list I hadn't pinned was boost-cpp. It doesn't matter much, as ARB uses only one header-only lib and the oldest conda-forge version 1.63 is plenty new enough.

Still, pinned it to be conforming.